Mu2eG4/src/ConstructMaterials.cc

```diff
@@ -1660,7 +1660,14 @@ namespace mu2e {
 
      BP -> AddElement(elB, 5*CLHEP::perCent);
      BP -> AddMaterial(Poly, 95*CLHEP::perCent);
+    }
 
+    mat = uniqueMaterialOrThrow( "SteelShot");
+    {
+      //7.85 g/cm3 is the density of steel and 63.5% is the densest packing of spheres
+      double density = 7.85 * .635;
```

Comment:
You followed the example code exactly, which is great.

I would like to start improving on this model as targets of opportunity.  When we first wrote this, constexpr did not exist - we should use it everywhere that we can and this is one place.   No need to change it in other places in this file - that's what I mean by targets of opportunity.

Reply:
Thanks for fixing it.